package.json

```diff
@@ -55,6 +58,7 @@
     "grunt-template-jasmine-requirejs": "0.2.3",
     "imports-loader": "0.6.5",
     "json-loader": "0.5.4",
+    "junit-report-builder": "1.1.1",
```

Comment:
I wasn't actually able to get it working with 3.9.3, but the instructions using 3.9.1 are linked here: https://oktawiki.atlassian.net/wiki/display/eng/Sign-in+Widget+Development